content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -510,7 +510,7 @@ Follow the steps given below to rollback the Deployment from the current version
 
     The output is similar to this:
     ```
-    deployment.apps/nginx-deployment
+    deployment.apps/nginx-deployment rolled back
```

Comment:
@efistasija
pls sign cla
see https://git.k8s.io/community/CLA.md#the-contributor-license-agreement